src/pages/categories-page/CategoryPage.js

```diff
@@ -0,0 +1,53 @@
+import { React } from "react";
+import { Footer } from "../../components";
+import "./categorypage.css";
+import { useQuiz } from "../../contexts";
+import { useParams, Link } from "react-router-dom";
+
+export function CategoryPage() {
+  const { quizState } = useQuiz();
+  const { categoryName } = useParams();
+  const getQuizList = () => {
+    if (categoryName === "All") {
+      return quizState.quizList;
+    }
+    const quizList = quizState.quizList.filter(
+      (quiz) => quiz.categoryName === categoryName
+    );
+    return quizList;
+  };
+
+  return (
+    <div>
+      <div className="quiz-category flex-row">
+        <div className="quiz-category-title">{categoryName} Quizzes</div>
+        <div className="flex-row quiz-category-list">
+          {getQuizList().map((quiz) => {
+            return (
+              <div className="card card-box-shadow">
```

Comment:
key prop is missing. While mapping the data, please add key prop to the parent element.

Reply:
done